src/main/java/uk/gov/companieshouse/pscstatementdataapi/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.companieshouse.pscstatementdataapi.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class ResourceNotFoundException extends ResponseStatusException {
+
+    String message;
+    public ResourceNotFoundException(HttpStatus status, String msg){
+        super(status, msg);
+        this.message = msg;
+    }
+
+    public String getMsg(){
```

Comment:
I'm a little surprised this is necessary, I thought getMessage was a part of most base exceptions. Is there any way to avoid setting the msg to message?